clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -42,7 +41,8 @@ const (
 	e2eSchedulingDurationMetricName           = model.LabelValue(schedulermetric.SchedulerSubsystem + "_e2e_scheduling_duration_seconds_bucket")
 	schedulingAlgorithmDurationMetricName     = model.LabelValue(schedulermetric.SchedulerSubsystem + "_scheduling_algorithm_duration_seconds_bucket")
 	frameworkExtensionPointDurationMetricName = model.LabelValue(schedulermetric.SchedulerSubsystem + "_framework_extension_point_duration_seconds_bucket")
-	schedulingLatencyMetricName               = model.LabelValue(schedulermetric.SchedulerSubsystem + "_" + schedulermetric.DeprecatedSchedulingDurationName)
+	bindingDurationMetricName                 = model.LabelValue(schedulermetric.SchedulerSubsystem + "_binding_duration_seconds_bucket")
```

Comment:
binding is already one of the extension points, so we don't really need a separate metric for it.